numpyro/distributions/discrete.py

```diff
@@ -165,6 +171,15 @@ def variance(self):
     def support(self):
         return constraints.integer_interval(0, self.total_count)
 
+    def enumerate_support(self):
```

Comment:
@eb8680 - Do we also need the `expand` keyword for `enumerate_support`?

Reply:
I did `grep` in Pyro and found no such usage case. :D

Reply:
I think you'll probably find it in the code for `enum_messenger`.

Reply:
> Do we also need the expand keyword for enumerate_support?

Not really, it should be fine to only implement `expand=False`.